app/controllers/settings.js

```diff
@@ -0,0 +1,20 @@
+import Controller from '@ember/controller';
+
+import { restartableTask } from 'ember-concurrency';
+import { service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class SettingsController extends Controller {
+  @service currentSession;
+  @tracked bestuurseenheidEmail;
+
+  get bestuurseenheid() {
+    return { email: this.bestuurseenheidEmail };
+  }
+
+  saveBestuurseenheidSettings = restartableTask(async () => {});
```

Comment:
this looks very strange why do we need it?

Reply:
Removed it completely, didn't know that you could leav out the argument of `onSave` in form::instance